my_game.py

```diff
@@ -7,14 +7,27 @@
 """
 
 import arcade
+import arcade.gui
 import math
 import random
 import tomli
+import tomli_w
+import os
 
 # load the config file as a dict
 with open('my_game.toml', 'rb') as fp:
     CONFIG = tomli.load(fp)
 
+# Load the saved settings file into the CONFIG dict
+try:
+    with open("my_game_edit.toml", "rb") as f:
```

Comment:
What is _my_game_edit.toml_?

Reply:
As it says in the comment above, it is the saved settings file. Should i change the name?